test/unit/helpers/views/ChoiceMessages.scala

```diff
@@ -22,9 +22,9 @@ trait ChoiceMessages {
 
   val arrivalDecLabel: String = "movement.choice.arrival.label"
   val departureDecLabel: String = "movement.choice.departure.label"
-  val associateDecLabel: String = "movement.choice.associate.label"
-  val disassociateDecLabel: String = "movement.choice.disassociateDucr.label"
-  val shutMucrLabel: String = "movement.choice.shutMucr.label"
+  val associateDecLabel: String = "movement.choice.associateucr.label"
```

Comment:
Maybe would be worth to drop this helper trait?

Reply:
YESSSSSS!!! Thanks! Done.